main/postgresql/APKBUILD

```diff
@@ -36,6 +36,11 @@ builddir="$srcdir/$pkgname-$pkgver"
 options="!checkroot"
 
 # secfixes:
+#   11.3-r0:
+#   - CVE-2019-10127
+#   - CVE-2019-10128
```

Comment:
CVE-2019-10127 and CVE-2019-10128 are Windows-only, so this package has never been affected by those.

https://www.postgresql.org/about/news/1939/

Reply:
It was commited as 7cf139bac41c8f2e1885d5f99334daeaeb059ac3 so I updated commit to remove wrong secfixes